configure_build_serve/requirements.txt

```diff
@@ -1 +1 @@
-ghga-service-chassis-lib==0.3.0
+ghga-service-chassis-lib==0.11.0
```

Comment:
Can we do ^0.11.0?

Reply:
I think this requirements.txt file has to be updated as the library updates by us. Since the update of the library is also under our control and we can live with the "=="